client/src/lib.rs

```diff
@@ -0,0 +1,177 @@
+use ed25519_dalek::{Keypair, PublicKey};
+use failure::{err_msg, Error};
+use hpos_state_core::admin_keypair_from;
+use serde::*;
+use wasm_bindgen::prelude::*;
+
+#[wasm_bindgen]
+pub struct HpAdminKeypair(Keypair);
+
+#[derive(Serialize, Deserialize, Debug)]
+struct Payload {
+    method: String,
+    uri: String,
+    body_string: String,
```

Comment:
```suggestion
    body: String,
```

Seems to be asymmetrical  to `method` and `uri` (why not `method_string` and `uri_string`?).

Reply:
We should match the RFC which says `method`, `request` and `body`

Reply:
will do